frontend/src/common/enums/recommitment-status.ts

```diff
@@ -61,3 +61,16 @@ export enum RecommitmentStatusLabel {
   // member will be set as INACTIVE in the particular program
   SUPERVISOR_NO_RESPONSE = 'No Supervisor Response',
 }
+
+export const RecommitmentStatusFilter = [
+  { value: 'ALL', label: 'All' },
+  { value: 'NEW', label: 'New' },
+  { value: 'ACTIVE', label: 'Active' },
+  { value: 'INACTIVE', label: 'Inactive' },
+  { value: 'MEMBER_COMMITTED', label: 'Recommitted' },
+  { value: 'MEMBER_DENIED', label: 'Member Declined - Not Returning' },
+  { value: 'MEMBER_NO_RESPONSE', label: 'Missed Recommitment' },
```

Comment:
We're going to need one more change here - "Missed Recommitment" needs to cover two cases: member no response and supervisor no response